pkg/util/util.go

```diff
@@ -41,12 +43,16 @@ type JobMergeData struct {
 	Action string `json:"action,omitempty"`
 }
 
-// InitializeTemplates initializes the temolates needed by this controller, it must be called at controller boot time
-func InitializeTemplates(jobTempateFileName string, cronJobTemplateFilename string) error {
-	text, err := ioutil.ReadFile(jobTempateFileName)
+// InitializeTemplates initializes the templates needed by this controller, it must be called at controller boot time
+func InitializeTemplates(jobTemplateFileName string, cronJobTemplateFileName string) error {
+	text, err := ioutil.ReadFile(jobTemplateFileName)
 	if err != nil {
-		log.Error(err, "Error reading job template file", "filename", jobTempateFileName)
-		return err
+		log.Info("Error reading job template file, using default one.", "filename", jobTemplateFileName)
+		text, err = ioutil.ReadFile(defaultJobTemplateFileName)
```

Comment:
When I'm providing explicit overrides for default templates, I'd prefer if the app loudly refused to start when an error was found in the templates, instead of quietly loading some completely different templates (in this case, the default ones). Would it be ok with you if we changed the logic here to just crash/stop in case of an error?

Reply:
Yes, you're right - the user would probably live in obliviousness to the fact that default configs have been loaded.